src/container/UserContainer.java

```diff
@@ -18,6 +18,12 @@ public static void save(User user) {
         users.add(user);
     }
 
+    public static void saveAll(List<User> listUsers) {
```

Comment:
don't use data structure in names. I mean set, list, etc.
today you have list, tommorow it might be set. In this case you need to change the name

just users